src/components/DateRangePickerInput.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   readOnly: PropTypes.bool,
   showCaret: PropTypes.bool,
   showDefaultInputIcon: PropTypes.bool,
+  inputIconPosition: PropTypes.oneOf('before', 'after'),
```

Comment:
I'm not sure you need to refactor much; newly importing the shape should be fine.